libtraceevent/parse-filter.c

```diff
@@ -1624,9 +1624,11 @@ int pevent_filter_clear_trivial(struct event_filter *filter,
 		case FILTER_TRIVIAL_FALSE:
 			if (filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
 		case FILTER_TRIVIAL_TRUE:
 			if (!filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
```

Comment:
@namhyung  I think `trivial filter` means it's type is `FILTER_ARG_BOOLEAN`,
and this `pevent_filter_clear_trivial()` function to just remove the only `trivial filters` in all filters.
So I think it is fine to use the fall-through comment and `break;` like above.

But I don't know where `pevent_filter_clear_trivial()` is called in uftrace source code and linux kernel source code..
So weird.